superset-frontend/src/components/Chart/ChartErrorMessage.tsx

```diff
@@ -42,6 +42,5 @@ export const ChartErrorMessage: React.FC<Props> = ({
     ...error,
     extra: { ...error.extra, owners },
   };
-
```

Comment:
nit: add this space back?

Reply:
Weird rebase stuff haha! Added back in [`this commit`](https://github.com/apache/superset/pull/20852/commits/61bdc42398bc3e8f458d1ba27536a4322b8e5097).